doc/howto/instances_access_files.md

```diff
@@ -0,0 +1,91 @@
+(instances-access-files)=
+# How to access files in an instance
+
+You can access the files from your instance and edit them, delete them, pull them to or push them from the host machine, or mount a file system from the instance to the host machine.
```

Comment:
This is quite a long sentence. Can we break it up a little to improve readability?
Also the "from the host machine" bit isn't quite right, as LXD can expose its services onto the network (via its API) so the `lxc` client command doesn't have to be run from the host machine.

How about something like:

> You can manage files inside an instance using the LXD client without needing to access the instance via the network.
> Files can be individually edited or deleted, pushed from or pulled to the local machine.
> Alternatively the instance's filesystem can be mounted onto the local machine.